### ISSUE TITLE
stoploss is not canceled on binance futures

### ISSUE THREAD

**Issue by hodlbe** [freqtrade/freqtrade#7838]
## Describe your environment

  * Operating system: Ubuntu 20.04.1
  * Python Version: 3.8.10 (`python -V`)
  * CCXT version: _____ (`pip freeze | grep ccxt`)
  * Freqtrade Version: 2022.10 (`freqtrade -V` or `docker-compose run --rm freqtrade -V` for Freqtrade running in docker)
  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

stoploss_on_exchange = true

binance futures exchange.
I have trailing and stoploss_on_exchange
Stop losses are constantly set on the exchange, but the old ones are not deleted. I hit my stop loss limit and the bot dies.

### Steps to reproduce:

  1. _____
  2. _____
  3. _____
  
### Observed Results:

  * What happened?
Old stoploss is not canceled

  * What did you expect to happen?
every time a new stop loss is set, the old one is canceled

### Relevant code exceptions or logs

Note: Please copy/paste text of the messages, no screenshots of logs please.

Max open order limit reached (the log was not saved, 50 stop orders were placed on the binance)

**Comment by xmatthias** [comment]
you'll have to provide logs for us to analyze this further - best for some period where the bot was creating "a few" new stoplosss. 

Please also make sure to provide these logs with
* an uptodate version (2022.11 is the current version)
* a trade that was opened with that version
* All dependencies uptodate - following the [updating guide](https://www.freqtrade.io/en/latest/updating/) for your installation method

**Comment by hodlbe** [comment]
> вам нужно будет предоставить нам журналы для дальнейшего анализа — лучше всего за какой-то период, когда бот создавал «несколько» новых стоп-лоссов.
> 
> Пожалуйста, также не забудьте предоставить эти журналы с
> 
> * актуальная версия (текущая версия 2022.11)
> * сделка, которая была открыта с этой версией
> * Все зависимости обновлены — следуйте инструкциям по [обновлению](https://www.freqtrade.io/en/latest/updating/) для вашего метода установки .

Update Freqtrade to 2022.11

Crushed.... (Error)

```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/freqtrade_main/freqtrade/freqtrade/exchange/exchange.py", line 1174, in stoploss
    order = self._api.create_order(symbol=pair, type=ordertype, side=side,
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/binance.py", line 2997, in create_order
    response = getattr(self, method)(self.extend(request, requestParams))
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/base/exchange.py", line 500, in inner
    return entry(_self, **inner_kwargs)
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/binance.py", line 5586, in request
    response = self.fetch2(path, api, method, params, headers, body, config, context)
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/base/exchange.py", line 2625, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/base/exchange.py", line 636, in fetch
    skip_further_error_handling = self.handle_errors(http_status_code, http_status_text, url, method, headers, http_response, json_response, request_headers, request_body)
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/binance.py", line 5564, in handle_errors
    self.throw_exactly_matched_exception(self.exceptions['exact'], error, feedback)
  File "/root/freqtrade_main/freqtrade/.env/lib/python3.8/site-packages/ccxt/base/exchange.py", line 2898, in throw_exactly_matched_exception
    raise exact[string](message)
ccxt.base.errors.BadRequest: binance {"code":-1106,"msg":"Parameter 'stopPrice' sent when not required."}

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/freqtrade_main/freqtrade/freqtrade/freqtradebot.py", line 1080, in create_stoploss_order
    stoploss_order = self.exchange.stoploss(
  File "/root/freqtrade_main/freqtrade/freqtrade/exchange/common.py", line 179, in wrapper
    raise ex
  File "/root/freqtrade_main/freqtrade/freqtrade/exchange/common.py", line 164, in wrapper
    return f(*args, **kwargs)
  File "/root/freqtrade_main/freqtrade/freqtrade/exchange/exchange.py", line 1196, in stoploss
    raise TemporaryError(
freqtrade.exceptions.TemporaryError: Could not place stoploss order due to BadRequest. Message: binance {"code":-1106,"msg":"Parameter 'stopPrice' sent when not required."}

```

**Another problem is that he is constantly looking for a deal that is already closed.**
lots of loop and loop 

```
2022-12-02 13:31:15,790 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Retrying still for 5 times.
2022-12-02 13:31:15,791 - freqtrade.exchange.common - INFO - Applying DDosProtection backoff delay: 1
2022-12-02 13:31:17,675 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Retrying still for 4 times.
2022-12-02 13:31:17,675 - freqtrade.exchange.common - INFO - Applying DDosProtection backoff delay: 2
2022-12-02 13:31:19,908 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Retrying still for 3 times.
2022-12-02 13:31:19,909 - freqtrade.exchange.common - INFO - Applying DDosProtection backoff delay: 5
2022-12-02 13:31:25,154 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Retrying still for 2 times.
2022-12-02 13:31:25,154 - freqtrade.exchange.common - INFO - Applying DDosProtection backoff delay: 10
2022-12-02 13:31:35,391 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Retrying still for 1 times.
2022-12-02 13:31:35,391 - freqtrade.exchange.common - INFO - Applying DDosProtection backoff delay: 17
2022-12-02 13:31:52,621 - freqtrade.exchange.common - WARNING - fetch_order() returned exception: "Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}". Giving up.
2022-12-02 13:31:52,622 - freqtrade.freqtradebot - WARNING - Error updating Order 4021351742 due to Order not found (pair: OCEAN/USDT id: 4021351742). Message: binance {"code":-2013,"msg":"Order does not exist."}.
2022-12-02 13:31:52,622 - freqtrade.freqtradebot - WARNING - Order is older than 5 days. Assuming order was fully cancelled.
2022-12-02 13:31:52,623 - freqtrade.freqtradebot - INFO - Buy order cancelled on exchange for Trade(id=4, pair=OCEAN/USDT, amount=16512.00000000, is_short=True, leverage=20.0, open_rate=0.13980000, open_since=closed).
2022-12-02 13:31:52,623 - freqtrade.freqtradebot - WARNING - Orderid for trade Trade(id=4, pair=OCEAN/USDT, amount=16512.00000000, is_short=True, leverage=20.0, open_rate=0.13980000, open_since=closed) is empty.

```

**Comment by xmatthias** [comment]
The exception (top of the exception) seems to be truncated - so it's missing some detail.

Based on the error - no stoploss should be placed on the exchange (it's aborting on the creation of the order).

**Comment by hodlbe** [comment]
> The exception (top of the exception) seems to be truncated - so it's missing some detail.
> 
> Based on the error - no stoploss should be placed on the exchange (it's aborting on the creation of the order).

Binance exchange, limit stop order previously worked yes and you can install them, after the update they broke.

in 2022.10 it worked, but the old ones were not deleted (there was an error in their number - the limit), now it stopped working.

**Comment by xmatthias** [comment]
The "not deleted" bug (in certain circumstances) was fixed in 2022.11. Obviously you'll need to make sure all these orders are (manually) canceled.

**Comment by xmatthias** [comment]
If i understood you right, then the original issue (stop orders not closing) has been fixed with the update.

If you encounter different issues with the updated version - please file a new issue so we can work on that independently, without having mixed in "what worked and what didn't in prior versions".
Please make sure to fill out the issue template - as well as providing the order_types configuration, and obviously the relevant logs.

`Order is older than 5 days. Assuming order was fully cancelled.` will be happening for orders in the database which were not properly marked as closed (`ft_is_open = 1`). it'll be a one-off action that will happen at startup, once it's through with these - it will save the result in the database and be done with it.
